Check broken links (#1354)
ocaml/v3.ocaml.org: data/planet/ocamlorg/turn-your-editor-into-an-ocaml-ide.md

handles the task of indenting your OCaml files.  It is an OCaml executable that
can be used from the command line or directly from your editor.

> The second tool, [merlin](http://the-lambda-church.github.io/merlin/ - [404 Not Found]) performs

Reviewer: ```suggestion
The second tool, [Merlin](https://github.com/ocaml/merlin) performs
```

Reviewer 2: ```suggestion
The second tool, [merlin](https://web.archive.org/web/20150101034910/http://the-lambda-church.github.io/merlin/) performs
```

> To maintain synchronization with the compiler, merlin needs some information

Reviewer: ```suggestion
To maintain synchronisation with the compiler, Merlin needs some information
```

about the structure of your project: build and source directories, package
dependencies, syntax extensions.  This structure can be described in a `.merlin` file in the root directory of your project.

> The [`.merlin` file for the merlin project](https://github.com/the-lambda-church/merlin/blob/master/.merlin - [404 Not Found]) illustrates the syntax.

Reviewer: ```suggestion
The [`.merlin` file for the Merlin project](https://github.com/ocaml/merlin/tree/master) illustrates the syntax.
```
Although I don't think the ReadMe illustrates the syntax... no `.merlin` file in the repo. Regardless, "Merlin" should be capitalized before "project."

Reviewer 2: ```suggestion
The `.merlin` file for the Merlin project illustrates the syntax.
```
